lib/hawkular/metrics/metric_api.rb

```diff
@@ -79,14 +79,15 @@ def query(tags = nil)
       # @param id [String]
       # @return [MetricDefinition]
       def get(id)
-        the_id = @client.hawk_escape id
```

Comment:
While I have no objection in general to replace `hawk_escape` by `url_encode` I think it should be done in one go and consistently.